modules/Neo4jDataAccess.py

```diff
@@ -40,9 +42,9 @@ def __init__(self, debug=False, neo4j_creds=None, batch_size=2000, timeout="60s"
         self.batch_size = batch_size
         self.tweetsandaccounts = """
                   UNWIND $tweets AS t
-                      //Add the Tweet
+                      //Add the Tweet 
```

Comment:
A lot of whitespace edits -- you may want to check your editor and start using `git add -p` to be more selective